src/greeter/KeyboardModel.h

```diff
@@ -46,6 +47,8 @@ namespace SDDM {
         KeyboardModel();
         virtual ~KeyboardModel();
 
+        void setProxy(GreeterProxy *proxy);
```

Comment:
If a member always needs to be set in order to not crash and is constant throughout the lifespan of the object it's better to pass it as an arg in the constructor.

Reply:
Made it so it's not mandatory. We don't yet have the GreeterProxy when we construct the KeyboardModel.